scripts/pages/zoning/zoning.ts

```diff
@@ -200,6 +200,8 @@ class ZoneMenuHandler {
 	}
 
 	initialize() {
+		this.panels.zoningMenu.updateEditorRegions([]);
```

Comment:
It's not possible for editor region entities to still exist after the map unloads/reloads. Clearing the regions like this is also already done explicitly by the game when the map loads. Does this change make a known difference?

Reply:
I added this because stale zones were drawn and never cleared on map reload.

For example, editing a track so all zones in that track are visible, reload map and select a checkpoint, the checkpoint doesn't blink/flash because the old fully-on zone was still rendered.

I can do another explicit test to see if this is till an issue with the rest of the changes.